packages/ee/billing/backend/billing.service.ts

```diff
@@ -22,13 +22,23 @@ function getDefaultPlanId(): string {
     if (stripeSecret.startsWith('sk_test')) {
         return "price_1MeoK3KZ0dZRqLEKXIsGoguO";
     } else {
-        return "price_1MfMlJKZ0dZRqLEKWHrxY97A";
+        return "price_1MgpQ4KZ0dZRqLEKqMiS8vrf";
```

Comment:
I think It will be better to give these ids some names and save them into constant vars instead of repeating them mysteriously everywhere